analyses/oncoprint-landscape/01-plot-oncoprint.R

```diff
@@ -0,0 +1,237 @@
+# This script displays an oncoprint displaying the landscape across PBTA given
+# the relevant metadata and output MAF files from the snv callers strelka2,
+# mutect2, lancet, and vardict. It addresses issue #6 in the OpenPBTA-analysis
+# github repository.
+#
+# Code adapted from the PPTC PDX Oncoprint Generation repository here:
+# https://github.com/marislab/create-pptc-pdx-oncoprints/tree/master/R
+#
+# Chante Bethell for CCDL 2019 and Jo Lynne Rokita
+#
+# #### USAGE
+# This script is intended to be run via the command line from the top directory
+# of the repository as follows:
+#
+# Rscript 'analyses/oncoprint-landscape/01-plot-oncoprint.R'
+
+#### Set Up --------------------------------------------------------------------
+
+# Install maftools
+if (!("maftools" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("maftools")
+}
+library(maftools)
+
+# Source the color palette for plots
+source(file.path("util", "oncoplot-palette.R"))
+
+#### Functions -----------------------------------------------------------------
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+#### Directories and Files -----------------------------------------------------
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper execution, no matter where
+# it is called from.
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Path to the data obtained via `bash download-data.sh`.
+data_dir <- file.path(root_dir, "data")
+
+# Path to output directory for plots produced
+plots_dir <-
+  file.path(root_dir, "analyses", "oncoprint-landscape", "plots")
+
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+
+genes <- NULL
+cnv_file <- NULL
+
+#### Command line options ------------------------------------------------------
+
+# Declare command line options
+option_list <- list(
+  optparse::make_option(
+    c("-m", "--maf_file"),
+    type = "character",
+    default = NULL,
+    help = "file path to MAF file that contains snv information",
+  ),
+  optparse::make_option(
+    c("-c", "--cnv_file"),
+    action = "store_true",
```

Comment:
My understanding is that you only want to use `action = "store_true"` if this is a logical ([`make_option` docs](https://www.rdocumentation.org/packages/optparse/versions/1.6.4/topics/make_option))